Core/Options.lua

```diff
@@ -1,4 +1,4 @@
-﻿local MogIt,mog = ...;
+local MogIt,mog = ...;
```

Comment:
Can't see what's changed here. Whitespace?

Reply:
Must be whitespace, maybe the line ending? My IDE does tend to clean up whitespace.